labgrid/driver/serialdriver.py

```diff
@@ -83,14 +83,19 @@ def get_export_vars(self):
             vars["protocol"] = self.port.protocol
         return vars
 
-    def _read(self, size: int = 1, timeout: float = 0.0):
+    def _read(self, size: int = 1, timeout: float = 0.0, max_size: int = None):
         """
         Reads 'size' or more bytes from the serialport
 
         Keyword Arguments:
         size -- amount of bytes to read, defaults to 1
+        max_size -- maximal amount of bytes to read, values 'None' or '0' do not restrict the read length,
```

Comment:
nitpick: this line shouldn't be longer than 100 characters.

Reply:
fixed this line with a line break, and another line in test_serialdriver.py that was too long